src/Text/makeStories.js

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
-import { text, select, boolean } from '@storybook/addon-knobs';
 import theme from '../theme';
 
-export default (storiesOf, { Text }) =>
+export default (storiesOf, { Text, View, Space }) =>
```

Comment:
Break this into multiple lines based on relevance.
Try to maintain the same order as the `.story.js` files.